src/scores/stats/tests/acovf.py

```diff
@@ -0,0 +1,135 @@
+"""
+Estimate autocovariances
```

Comment:
Thanks greatly for pulling this in. Test coverage is 82%. Please add more tests.